.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
```

Comment:
I added GHA, because Travis on my fork had some Maven repo problems. There was a workaround AFAIK, but I didn't want to deal with Travis too much.